src/resolvers/git.cr

```diff
@@ -363,7 +366,10 @@ module Shards
 
     private def valid_repository?
       command = "git config --get remote.origin.mirror"
-      Log.debug { command }
+      Log.with_context do
+        Log.context.set package: name
+        Log.debug { command }
```

Comment:
My small discovery.

I found `emit` could add context inplace (e.g `Log.warn exception: e, &.emit("Oh no!", user_id: 42)`).
The normal Log does not have this feature.

For example would be nice to have consistency:
```crystal
  Log.debug(exception: e, package: name) { "Oh no!" }
  Log.debug(exception: e, package: name) &.emit("Oh no!")
```